vpr/src/base/SetupVPR.cpp

```diff
@@ -609,6 +609,7 @@ static void SetupPlacerOpts(const t_options& Options, t_placer_opts* PlacerOpts)
     PlacerOpts->place_agent_algorithm = Options.place_agent_algorithm;
     PlacerOpts->place_constraint_expand = Options.place_constraint_expand;
     PlacerOpts->place_constraint_subtile = Options.place_constraint_subtile;
+    PlacerOpts->floorplan_split = Options.floorplan_split;
```

Comment:
change options to just be the number of cutlines

Reply:
or just call it number of bins, that matches my cases - number of horizontal partitions, number of vertical partitions